C++ programs/bubblesort.cpp

```diff
@@ -0,0 +1,37 @@
+// C++ program for implementation
+// of Bubble sort
+#include <bits/stdc++.h>
+using namespace std;
+
+// A function to implement bubble sort
+void bubbleSort(int arr[], int n)
+{
+	int i, j;
+	for (i = 0; i < n - 1; i++)
+
+		// Last i elements are already
+		// in place
+		for (j = 0; j < n - i - 1; j++)
+			if (arr[j] > arr[j + 1])
+				swap(arr[j], arr[j + 1]);
```

Comment:
where is swap function defined in this?

Reply:
The inbuilt swap function is used and for that header file is included.